install.sh

```diff
@@ -4,9 +4,9 @@
 set -e
 
 #chromebrew directories
-OWNER="skycocker"
-REPO="chromebrew"
-BRANCH="master"
+OWNER="${CREW_OWNER:-skycocker}"
+REPO="${CREW_REPO:-chromebrew}"
+BRANCH="${CREW_BRANCH:-master}"
```

Comment:
Should we add these new constants to the list?

Reply:
This would be good for testing, but crew update just ends up using https://github.com/skycocker/chromebrew/blob/9e4afa7874826f3942438cd8d7d1c9a781c2b298/crew#L391
```
    system "git fetch origin master"
    system "git reset --hard origin/master"
```
so we should probably have crew save the owner/rep/branch info in prefs too.